### PR TITLE
104 ta ms add shortlist open time

### LINKED ISSUE #104: TA-Add start and end time for shortlist creation
Add start and end time inputs to cohort form, so the admin can set times when shortlist creation for mentees is open

### PR DESCRIPTION
**Pull request by MayaS1111**
- Added columns to Cohort table
     - shortlist_start_time
     - shortlist_end_time
- Created shortlist_creation_open? method for cohort model
- Shortlist view displays if shortlist creation is open
     - Time adjusts for time zones
- Updated cohort form in admin view
     - Added start and end shortlist fields
     - Added required meetings field
     - Made program id, creator id, and contact id hidden
     
closes #104

### CODE REVIEW

File: app/models/cohort.rb
Comment (brvarner) [inline app/models/cohort.rb:1]: Love the added method! Also great stuff added to the model!

File: db/migrate/20241027210603_add_shortlist_start_and_end_time_to_cohort.rb
Comment (brvarner) [inline db/migrate/20241027210603_add_shortlist_start_and_end_time_to_cohort.rb:1]: Love a migration lol

**Review by brvarner — APPROVED** [review]
Great work per usual! Glad to see you using the models the way they're intended and adding methods to make our lives easier.

**Review by rezahusain — APPROVED** [review]
Logic looks really good Maya! Approved :)

**Pull request merged**